source/PowerAdjusterView.mc

```diff
@@ -110,19 +111,24 @@ class DataField extends Ui.SimpleDataField {
     const DURATION = Application.getApp().getProperty("duration").toNumber();
     const SLOPE = new Slope(Application.getApp().getProperty("slope"));
     const ALTPOWER = Application.getApp().getProperty("altPower_prop");
+    const PURE_POWER = Application.getApp().getProperty("purePower_prop");
     const HOMEALT = Application.getApp().getProperty("homeElevation_prop").toNumber();
     var homealt_factor = 1;
     var power_array = new [DURATION];
     var power_array_next_index = 0;
     var power_sum = 0;
     var power_array_complete = false;
+    var bikePower;
+    var bikePowerListener;
 
     // Constructor
     function initialize() {
         //Sys.println(POWER_MULTIPLIER);
         //Sys.println(Application.getApp().getProperty("slope"));
+        bikePowerListener = new AntPlus.BikePowerListener();
+        bikePower = new AntPlus.BikePower(bikePowerListener);
         Ui.SimpleDataField.initialize();
-        label = "adjPwr. " + DURATION.toString() + "s" + (ALTPOWER ? ",a" : "");
+        label = "adjPwr. " + DURATION.toString() + "s" + (ALTPOWER ? ",a" : "") + (PURE_POWER ? ",p" : "");
```

Comment:
Those who have quite a small screen would appreciate less commas. Like this:
var modifiers = (ALTPOWER ? "a" : "") + (PURE_POWER ? "p" : "");
label = "adjPwr. " + DURATION.toString() + "s" + (modifiers.length() > 0 ? "," + modifiers : "");